localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -329,6 +329,36 @@
         <trans-unit id="msgConnecting">
             <source xml:lang="en">Connecting to server "{0}" on document "{1}".</source>
         </trans-unit>
+        <trans-unit id="msgConnectionNotFound">
```

Comment:
I'm not sure about adding essentially debug messages into the output pane...but it seems fine to throw in there to make it easier to track down issues and if it gets to be too spammy later on we can change it I guess.